addons/godot-firebase/auth/auth.gd

```diff
@@ -388,14 +406,18 @@ func remove_auth() -> void:
     else:
         Firebase._printerr("No encrypted auth file exists")
 
+
 # Function to check if there is an encrypted auth data file
 # If there is, the game will load it and refresh the token
 func check_auth_file() -> void:
     var dir = Directory.new()
     if (dir.file_exists("user://user.auth")):
+        # Will ensure "auth_request" emitted
         load_auth()
     else:
         Firebase._printerr("No encrypted auth file exists")
+        emit_signal("auth_request", "File does not exist", auth)
```

Comment:
Same as above: we have to mention it's an auth file here so we don't potentially confuse people.